src/dmd/parse.d

```diff
@@ -3684,6 +3684,11 @@ final class Parser(AST) : Lexer
             check(TOK.rightParentheses);
             break;
 
+        case TOK.else_:
+            error("basic type expected, not `else`. Hint: there's no `static else`, use `else` instead.");
```

Comment:
Perhaps the hint should be a supplemental message.

Reply:
Should I strip the suffix `Hint:` then?

Reply:
Not sure.